### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by JeanDamien**

### CODE REVIEW

File: techio.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title : PHP
+plan:
+  - title: PHP on Tech.io
+    statement: markdowns/welcome.md
+projects:
+  first-project:
+    root: /php-template
+    runner:
+      name: php
```
Comment (Sy1v4in) [inline techio.yml:9]: We don't use the `techio` image runner?

File: php-template/UniverseTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php
+    include 'Universe.php';
+    function test() {
+        if (countAllStars([2,3]) ==  5 && countAllStars([9,-3]) == 6) {
+
+            echo("TECHIO> success true \r\n");
+            $file = file_get_contents("./Universe.php");
```
Comment (Sy1v4in) [inline php-template/UniverseTest.php:7]: No other way?

File: techio.yml (hunk 1 above)
Comment (JeanDamien) [reply techio.yml:9]: Don't we promote the use of official image ?

File: php-template/UniverseTest.php (hunk 2 above)
Comment (JeanDamien) [reply php-template/UniverseTest.php:7]: Maybe we can use rename_function/override_function but we will need to install PECL package. I'll let @sidsharma27 answer that.
_Ps: PECL is deprecated now :/_

File: techio.yml (hunk 1 above)
Comment (Sy1v4in) [reply techio.yml:9]: Don't know?

**Pull request merged**